helm/sealed-secrets/templates/servicemonitor.yaml

```diff
@@ -3,19 +3,19 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: {{ include "sealed-secrets.fullname" . }}
-  {{- if .Values.serviceMonitor.namespace }}
-  namespace: {{ .Values.serviceMonitor.namespace }}
+  {{- if .Values.metrics.serviceMonitor.namespace }}
+  namespace: {{ .Values.metrics.serviceMonitor.namespace }}
   {{- else }}
   namespace: {{ include "sealed-secrets.namespace" . }}
   {{- end }}
   labels: {{- include "sealed-secrets.labels" . | nindent 4 }}
-    {{- if .Values.serviceMonitor.labels }}
-    {{- include "sealed-secrets.render" ( dict "value" .Values.serviceMonitor.labels "context" $) | nindent 4 }}
+    {{- if .Values.metrics.serviceMonitor.labels }}
```

Comment:
This is not documented in `values.yaml` or the README :)

Reply:
Done in https://github.com/bitnami-labs/sealed-secrets/pull/635.